src/List/ListItemText.js

```diff
@@ -20,6 +20,11 @@ export const styleSheet = createStyleSheet('MuiListItemText', () => {
         paddingLeft: 56,
       },
     },
+    dense: {
+      '& > *': {
```

Comment:
We have a `colorInherit` property on the Text component. We could imagine a `fontSizeInherit` property.